src/polymer/polymer-element-scanner.ts

```diff
@@ -159,8 +160,12 @@ class ElementVisitor implements Visitor {
               this.document, elementObject, 'callExpression');
           element.warnings = element.warnings.concat(parseResult.warnings);
           let expressionText = undefined;
-          if (elementObject.type === 'Literal') {
-            expressionText = elementObject.raw;
+          if (babel.isLiteral(elementObject)) {
+            // TODO(usergenic): How do I get equivalent of `raw` from Babylon
```

Comment:
issue

Reply:
Ah yes I forgot about this one.  We have zero test-coverage for the `expressionText` returned in the observers set, so I'm not sure what we're expecting here.  (I tried changing the return value to 'potato' and everything still passed...)

@rictic @justinfagnani can you weigh in on what the intent of `expressionText` is before I file the issue and/or add test and correct this bit?

Reply:
While tests don't cover, I ran experiments and found out `.raw` is essentially just the excerpt of the document from start to end.  So, I modified elementObject.raw to be this.document.contents.slice(elementObject.start, elementObject.end) and that returned identical content.  No need to file issue as this was the only occurrence of `.raw` being used anyways.

Reply:
This looks correct to me. If there's an issue it'll come up in the linter's tests when we update it to analyzer @3.0.0 and we can fix it then, but this is very likely correct.